src/library/scala/collection/IterableOnce.scala

```diff
@@ -1045,6 +1046,8 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
   /** Displays all elements of this $coll in a string using start, end, and
     *  separator strings.
```

Comment:
I don't know if odd indentation matters to scaladoc? I think at one time it had a bug, maybe.